src/lib.rs

```diff
@@ -116,6 +120,22 @@ impl AnalysisLoader for CargoAnalysisLoader {
     }
 }
 
+fn host_triple() -> String {
```

Comment:
So, we should be providing the target triple here, not the host triple. I think there is no nice way to do that right now (long term we should get this from Cargo and pass it in from the RLS). So using the host triple and not supporting cross-compilation is fine for now.

Can you rename this to `target_triple` and add a comment that we are using the host triple as a temporary hack?

I think it would be better to parse the host triple from the sys root string, rather than calling rustc.